test/src/tests-mparticle-instance-manager.ts

```diff
@@ -450,74 +440,59 @@ describe('mParticle instance manager', () => {
                 5
             );
 
-            // setTimeout to allow config to come back from the beforeEach initialization
+            await waitForCondition(hasConfigurationReturned);
```

Comment:
Same here. Why is this going before the logging function?

Reply:
same answer